### DISCUSSION TITLE
Add URL directlly in web feed if item._microfeed.is_external_url

### DISCUSSION THREAD

**Discussion by gabbello** [microfeed/microfeed#26] [Ideas] [answered]
Maybe one idea would be to add the external url directly in the web feed list if the item is of type external_url.

**Comment by wenbinf** [comment, marked as answer]
You can do this by editing the Web Feed's html code - 
* Go to /admin/settings/code-editor/?type=themes&theme=custom#webFeed

Or you can find it in Settings:

<img width="1015" alt="Screenshot 2023-01-03 at 8 09 23 AM" src="https://user-images.githubusercontent.com/1719237/210283952-ec2ee362-61bd-4464-8123-d6d674a8205e.png">


* Then put code in it:
```
{{#items}}
    <div>
        <h3>            
            <a href="{{#attachments}}{{url}}{{/attachments}}">{{title}}</a>            
        </h3>
    </div>
{{/items}}
```

TODO for us: 
* Add tutorials / docs for updating html/css code
* Support multiple themes
* Provide a few default themes (e.g., curated list, instagram-like feed...)

**Comment by gabbello** [comment]
Thanks, it works now. I've actually tried to add it myself before, but missed the exact semantics.
